recipes/_install-linux.rb

```diff
@@ -89,5 +89,13 @@
       action package_action # default is :install
       allow_downgrade node['datadog']['agent_allow_downgrade']
     end
+  when 'suse'
+    zypper_package 'datadog-agent' do
```

Comment:
One more thing: foodcritic fails on one of the resource's attributes. Based on the docs (https://docs.chef.io/resource_zypper_package.html) `allow_downgrade` doesn't exist on this resource. Maybe passing a specific option in `options` instead could work?

Reply:
allow downgrade isn't a thing on suse, you're right. I believe chef allows you to downgrade on it without the flag, just by specifying a version.